build/gulp/tasks/stats.ts

```diff
@@ -124,9 +134,26 @@ task(
   ),
 )
 
+function readSummaryPerfStats() {
+  return _.chain(require(paths.perfDist('result.json')))
+    .mapKeys((value, key) => _.camelCase(key))
+    .mapValues(result => ({
+      actualTime: _.omit(result.actualTime, 'values'),
+      baseTime: _.omit(result.baseTime, 'values'),
+    }))
+    .value()
+}
+
+function readCurrentBundleStats() {
+  return _.mapKeys(require(paths.docsSrc('currentBundleStats.json')), (value, key) =>
```

Comment:
we could make this to be a constant (declared by this module), to avoid repetition